test/GoodGhostingGasEstimate.test.js

```diff
@@ -156,5 +158,21 @@ contract("GoodGhosting", (accounts) => {
                 }, "unable to withdraw amount");
             }
         });
+
+        it("admin is able to withdraw the pool fee collected", async () => {
+            if (configs.deployConfigs.customFee !== 0) {
+                const result = await goodGhosting.adminFeeWithdraw({ from: admin });
+                truffleAssert.eventEmitted(
+                    result,
+                    "AdminWithdrawal",
+                    (ev) => {
+                        const adminFee = (new BN(configs.deployConfigs.customFee).mul(ev.totalGameInterest).div(new BN('100')));
+                        return adminFee.lte(ev.amount);
+                    })
+                await truffleAssert.reverts(goodGhosting.adminFeeWithdraw({ from: admin }), "Admin has already withdrawn");
+            } else {
+                await truffleAssert.reverts(goodGhosting.adminFeeWithdraw({ from: admin }), "No Fees Earned");
+            }
+        })
```

Comment:
Really minor, but for this gas estimation test unit, we should keep tests separate:
- one for a successful admin fee withdrawal
- one for a revert if trying to withdraw more than once
- one for a revert if no fees earned


by mixing all the tests, we'll get more inaccurate gas estimations.